tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "importHelpers": true,
     "inlineSources": true,
     "inlineSourceMap": false,
-    "lib": ["dom", "ES2020"],
+    "lib": ["dom", "ES2022"],
```

Comment:
What's the benefit of making this change?  Are there new JS APIs that we want to use? 

Also, does this mean that browsers need to support ES2022 to run this polyfill?

Reply:
We need this in order for TS to type-check / recognize the Intl.supportedValuesOf API - without this, TS complains that it doesn't exist on the Intl namespace.

Reply:
AFAICT, this just changes the types available to TS, and it doesn't change the runtime requirements any more than we already have by using Intl.supportedValuesOf in the polyfill sources (which seems fairly new? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf).

Reply:
Actually, having re-read the code we do fallback if the API isn't available, so I don't think we've bumped any runtime requirements here.